src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -201,4 +201,7 @@
   <data name="WindowStyleArgumentNotImplemented" xml:space="preserve">
     <value>Parameter -WindowStyle is not implemented on this platform.</value>
   </data>
+  <data name="MissingWorkingDirectoryArgument" xml:space="preserve">
+    <value>Cannot process the command because -WorkingDirectory requires a directory path. Supply a path for the -WorkingDirectory parameter and then try the command again.</value>
```

Comment:
Is the 2nd part of the sentence really necessary? To me it does not add more valuable information than what the first sentence already states.

Reply:
I borrowed it as a template from another message.  I'll make it more brief and update the other one as well.